pagopa/io-app
# feat: [IOPLT-341] Rework e2e test job with matrix

## Short description
This PR refactors the e2e workflow to run with a matrix strategy

## List of changes proposed in this pull request
- Reworks e2e test workflow

## How to test
Check the [latest run of the job to verify](https://github.com/pagopa/io-app/actions/runs/8378196304?pr=5616)
